tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -24,9 +24,18 @@
 
 from copy import deepcopy
 
+try:
+    from unittest import mock
+except ImportError:
+    try:
+        import mock
+    except ImportError:
+        mock = None
```

Comment:
Asking to learn: how does this play with the `from mock import ...` below?

Reply:
Good question. This is a different PR, but we should make the way we import `mock` consistent, because I see two ways of doing it:
```
try:
    from unittest import mock
except ImportError:
    try:
        import mock
    except ImportError:
        mock = None
```
or just:
```
import mock
```

Reply:
Yes what I meant is that your have both a top level module import and a `from mock import ...` below.  I usually do one or the other.

Reply:
Ah yes, I see now. I've removed the new import completely because it isn't required as you mentioned. Beside that I've also created a new ticket of cleaning up the way we import mock:
https://issues.apache.org/jira/browse/AIRFLOW-1596?jql=project%20%3D%20AIRFLOW